app/src/main/java/com/kickstarter/libs/CurrentUserV2.kt

```diff
@@ -84,21 +84,16 @@ class CurrentUserV2(
     private val userPreference: StringPreferenceType
 ) : CurrentUserTypeV2() {
     val user = BehaviorSubject.create<KsOptional<User>>()
+
     init {
-        user
-            .filter { it.isPresent() }
-            .map { it.getValue() }
-            .skip(1)
-            .filter { `object`: User? -> `object`.isNotNull() }
-            .subscribe { u: User? ->
```

Comment:
Subscription to the internal `user` observable is no longer needed. The user is persisted in `Login()` method or `refresh()` method